ipi/engine/properties.py

```diff
@@ -2934,9 +2934,9 @@ def __init__(self):
                        with the specified bead (with the level of discretization of the component), otherwise the 
                        centoid force is computed. The weight of the potential is not applied. """,
                 "func": lambda index, bead="-1": (
-                    self.system.forces.forces_component(int(index), False, False).sum(
-                        axis=0
-                    )
+                    self.system.forces.forces_component(
+                        int(index), weighted=False, interpolate=True
+                    ).sum(axis=0)
```

Comment:
Set interpolate to false, remove the division by nbeads (that should be for the component, and not the global one) and instead use mean